### ISSUE TITLE
Provide MacOS icon template for Inkscape

### ISSUE THREAD

**Issue by maxtron95** [elrumo/macOS_Big_Sur_icons_replacements#839]
Currently on the site, there is only templates for Sketch, Photoshop, Illustrator, and Figma. Is it possible to provide a MacOS Big Sur Icon template for Inkscape (.svg) users.

**Comment by hazarek** [comment]
I was going to open a title about this exact subject. Non-FOSS template files (.sketch, .ai, .psd, .fig) (viral ad). are completely against the Open Source and GNU spirit. Templates should be shared as SVG, EPS, PDF, ORA, PNG

**Comment by creepertron95** [comment]
May as well have GIMP .xcf also. Not all of us can pay for the Proprietary software.

**Comment by hazarek** [comment]
> May as well have GIMP .xcf also. Not all of us can pay for the proprietary software. (Turns out I can open the .ai in Inkscape, but it's a mess to deal with.)

Krita can import [PSD](https://docs.krita.org/en/general_concepts/file_formats/file_psd.html) 

> .psd, unlike actual interchange formats like *.pdf, *.tiff, *.exr, *.ora and *.svg doesn’t have an official spec online. Which means that it needs to be reverse engineered. Furthermore, as an internal file format, it doesn’t have much of a philosophy to its structure, as it’s only purpose is to save what Photoshop is busy with, or rather, what all the past versions of Photoshop have been busy with. This means that the inside of a PSD looks somewhat like Photoshop’s virtual brains, and PSD is in general a very disliked file-format.

**Comment by creepertron95** [comment]
cc @elrumo

**Comment by hazarek** [comment]
@creepertron95 I determined the measurements. here is a 3-line template
https://gist.github.com/hazarek/52704b2bc358ec93f766cff1bcc6959d

```
<svg width="1024" height="1024" xmlns="http://www.w3.org/2000/svg" style="background-color:darkgray">
  <rect rx="188" id="svg_1" height="824" width="824" y="100" x="100" fill="lightgray"/>
</svg>
```

**Comment by creepertron95** [comment]
@hazarek Thank you so much for the svg! Now all I need to do is to determine how to imitate drop shadows and other effects to the icons. The templates should be improved if that is the case.

**Comment by hazarek** [comment]
> @hazarek Thank you so much for the svg! Now all I need to do is to determine how to imitate drop shadows and other effects to the icons. The templates should be improved if that is the case.

Maybe I'll share with drop shadow+emboss effect tomorrow. I can't using inkscape on macbook(gesture doesn't work, [very slow and laggy](https://gitlab.com/inkscape/inkscape/-/issues/1614)). can't help.

**Comment by hazarek** [comment]
Revision;
BigSur App icon SVG template with official_shadow parameters
https://gist.github.com/hazarek/d95fde8078bafc53ddd8ebd5bb01e187

**Comment by creepertron95** [comment]
@hazarek We should make a Gimp/Krita version along with the svg version of the template as well, and ask @elrumo to include them on the site.

**Comment by hazarek** [comment]
> @hazarek We should make a Gimp/Krita version along with the svg version of the template as well, and ask @elrumo to include them on the site.

gimp and krita are not exchange formats. Any software can open svg. gimp and krita can import svg.

**Comment by creepertron95** [comment]
Can they import them in layers for them to be worked on?

**Comment by hazarek** [comment]
> Can the svg be imported in layers for them to be worked on in program?

yes but if the software supports it. vector editors can open svg / xml elements (groups) as layers.
i know these can open (illustrator, inkscape, affinity designer)
<img width="301" alt="Ekran Resmi 2020-11-27 02 34 56" src="https://user-images.githubusercontent.com/36481442/100397206-81a18180-3059-11eb-93f7-af468020b5fc.png">

**Comment by hazarek** [comment]
Unfortunately, open source software does not make designers happy.

**Comment by Jak3diQori** [comment]
I made this minimal MacOs Big Sur style Inkscape icon. 
I hope you like it!
![inkscape-mac](https://user-images.githubusercontent.com/61330999/115062547-c172e380-9eea-11eb-8b95-da3e62eda80e.png)

**Comment by Avtronboy** [comment]
> I made this minimal MacOs Big Sur style Inkscape icon.
> I hope you like it!
> ![inkscape-mac](https://user-images.githubusercontent.com/61330999/115062547-c172e380-9eea-11eb-8b95-da3e62eda80e.png)

Hi there, do you have the .icns file?

**Comment by elrumo** [comment]
> I made this minimal MacOs Big Sur style Inkscape icon.
> I hope you like it!
> ![inkscape-mac](https://user-images.githubusercontent.com/61330999/115062547-c172e380-9eea-11eb-8b95-da3e62eda80e.png)



> > I made this minimal MacOs Big Sur style Inkscape icon.
> > I hope you like it!
> > ![inkscape-mac](https://user-images.githubusercontent.com/61330999/115062547-c172e380-9eea-11eb-8b95-da3e62eda80e.png)
> 
> Hi there, do you have the .icns file?


Thanks for the icon @Jak3diQori! I've just uploaded it to the website, @Avtronboy you can find the icon here: https://macosicons.com/Inkscape

**Comment by Avtronboy** [comment]
Thanks! I love macOSicons.com! Sorry about the hack by the way!

Charles Blair---
